src/sentry/static/sentry/app/views/organizationIntegrations/integrationItem.jsx

```diff
@@ -22,7 +34,7 @@ export default class IntegrationItem extends React.Component {
         <Box>
           <IntegrationIcon size={compact ? 26 : 32} integration={integration} />
         </Box>
-        <Labels compact={compact}>
+        <this.labels compact={compact}>
```

Comment:
imo, `this.labels` is kind of inconsistent looking here, looks nicer if it were a component name.

Why not something in between
```
const Labels = p => {	
  const {compact, ...props} = p;	
  return compact ? <Flex align="center" direction="row" pl={1} {...props} /> : <Flex direction="column" pl={1} {...props} />;
};
```

or

```
const Labels = p => {	
  const {compact, ...props} = p;
  const flexProps = compact ? {align: 'center', direction: row, pl: 1} : {direction: 'column', pl: 1};
  return <Flex {...flexProps} {...props} />;
};
```

Reply:
just out of curiosity, are we trying to move away from things like

`const labels = ({compact, ...props}) => {}`

I'll stop writing it that way if so.

Reply:
ah no I was just copy and pasting, I do like destructuring like that.